apps/managedidentity/managedidentity.go

```diff
@@ -206,6 +216,7 @@ func (client Client) getTokenForRequest(req *http.Request) (accesstokens.TokenRe
 	}
 	var r accesstokens.TokenResponse
 	err = json.Unmarshal(responseBytes, &r)
+	r.GrantedScopes.Slice = append(r.GrantedScopes.Slice, req.URL.Query().Get(resourceQuerryParameterName))
```

Comment:
I suppose this is a reasonable assumption when the response doesn't provide a value, but shouldn't we prefer the response's `resource`?

Reply:
We discussed with @bgavrilMS about this, 
We don't read `resource` from response, and we assume that those will always be same.